### ISSUE TITLE
screen reader does not read column headers when moving through rows

### ISSUE THREAD

**Issue by frank1russo** [SitePen/dgrid#753]
This issue is caused by having the column headers and each row in their own <table> structures. For true accessibility, the data in the grid cells need to be associated with the headers, so we're having trouble meeting our accessibility requirements.

Is there a plan to fix this soon, or is there a workaround?

**Comment by sudheerravipati** [comment]
Hi 
Can you share the javascript code for this. I am using pq grid and in my case screen reader is not reading headers as header is in one table and data is in another table.

**Comment by Jianxia** [comment]
Same here. Is there any update on the status of the issue?
